src/GifPickerReact.css

```diff
@@ -28,7 +43,7 @@
     --gpr-category-list-padding: var(--gpr-horizontal-padding);
     --gpr-category-border-radius: 4px;
     --gpr-category-background-opacity: 40%;
-    --gpr-category-font-color: var(--gpr-picker-border-color);
+    --gpr-category-font-color: var(--gpr-cat-font-color);
```

Comment:
No idea what that does, `--gpr-cat-font-color` is not specified anywhere

Reply:
Updated was a workaround